### PR TITLE
Introduce timeout while deploying connector

### PR DESCRIPTION
**Pull request by wadhwa1**
# Description
Introduce timeout while deploying connector.

Fixes # (issue)
https://confluentinc.atlassian.net/browse/RCCA-11106

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
https://jenkins.confluent.io/job/cp-ansible-on-demand/1178/

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: plugins/modules/kafka_connectors.py [hunk 1]
```diff
@@ -217,6 +224,7 @@ def run_module():
     module_args = dict(
         connect_url=dict(type='str', required=True),
         active_connectors=dict(type='list', elements='dict', required=True),
+        timeout=dict(type='int', required=False, default=60),
```
Comment (utkarsh5474) [inline plugins/modules/kafka_connectors.py:227]: Default value here and for the var are diff. Is that intended?

**Review by utkarsh5474 — APPROVED** [review]
Thanks! LGTM
One minor comment.
Also, have we checked with customer whether this approach solves the issue they're having?

**Comment by wadhwa1** [comment]
> Thanks! LGTM One minor comment. Also, have we checked with customer whether this approach solves the issue they're having?

Thanks @utkarsh5474 ! 
yes, checked with customer, adding timeout=30 works for them and connector deployment was successful.

File: plugins/modules/kafka_connectors.py (hunk 1 above)
Comment (wadhwa1) [reply plugins/modules/kafka_connectors.py:227]: I had set it to 60 initially, but after checking with customer made it 30 at other place, forgot to update it here. thanks for pointing out :)

**Pull request merged**